projects/spidermonkey/build.sh

```diff
@@ -36,3 +36,8 @@ cd build_DBG.OBJ
 make "-j$(nproc)"
 
 cp dist/bin/js $OUT
+
+# Copy libraries.
+mkdir -p $OUT/lib
+cp -L /usr/lib/x86_64-linux-gnu/libc++.so.1 $OUT/lib
```

Comment:
I think we don't actually test running projects with engine "none" in CI, so this might not work, but I guess it's worth trying.